### PR TITLE
[test] Fixes prepare.py typo

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -60,6 +60,11 @@
         "option.s3url": "s3://djl-llm/opt-13b/",
         "option.tensor_parallel_degree": 2,
         "option.task": "text-generation"
+    },
+    "opt-30b": {
```
Comment (lanking520) [inline tests/integration/llm/prepare.py:64]: we cannot add this since our testing is on g5 12x. Memory is not enough

Comment (frankfliu) [reply tests/integration/llm/prepare.py]: removed

**Review by lanking520 — APPROVED** [review]
